src/products/views/ProductList/ProductList.tsx

```diff
@@ -192,6 +192,21 @@ export const ProductList: React.FC<ProductListProps> = ({ params }) => {
     params
   });
 
+  useEffect(() => {
+    const sortWithQuery = ProductListUrlSortField.rank;
```

Comment:
Can we move this function to a named one, so it's clear right away what is being done on query change?